examples/channels-workers/channels_workers.out

```diff
@@ -0,0 +1,9 @@
+//At the command line, navigate to the directory that contains the
+//`.bal` file and run the `ballerina run` command.
```

Comment:
Minor correction:
"//At the command line, navigate to the directory that contains the
//`.bal` file, and then run the `ballerina run` command."